src/client/components/data-table/index.pug

```diff
@@ -37,7 +37,7 @@ table.rwd-table
         +if('hasAddNewButton')
             tr
                 td(colspan="{{ columnList.length + 1 }}")
-                    row
+                    .row
```

Comment:
I realized that the row was rendered as <div class="row"... but wasn't sure how, changed it by thinking that it might cause future problems... 
Thanks for the explanation.